### DISCUSSION TITLE
Could I change the output (svg) font size?

### DISCUSSION THREAD

**Discussion by FFengIll** [badges/shields#5672] [Q&A] [answered]
If I get the output as SVG, I have no idea to change the font size (no query support).
Could I get a way to do it ?

**Comment by chris48s** [comment, marked as answer]
You can make the whole badge bigger or smaller by scaling it when you embed e.g:

`<img src="https://img.shields.io/badge/build-passing-%20brightgreen" />`
<img src="https://img.shields.io/badge/build-passing-%20brightgreen" />


`<img src="https://img.shields.io/badge/build-passing-%20brightgreen" height="50" />`
<img src="https://img.shields.io/badge/build-passing-%20brightgreen" height="50" />


`<img src="https://img.shields.io/badge/build-passing-%20brightgreen" height="10" />`
<img src="https://img.shields.io/badge/build-passing-%20brightgreen" height="10" />

but there's no way to make just the text bigger/smaller but keep the badge the same size. The dimensions of the badge are relative to the size of the text.

**Reply by imprithwishghosh** [reply]
how do I change on this link : ![](https://komarev.com/ghpvc/?username={username})

like how would I be able to increase the size of this badge ???? without adding img src

**Reply by rnons** [reply]
the max height is 20 even when 50 is specified?

<img width="1274" height="138" alt="image" src="https://github.com/user-attachments/assets/6436bdaf-a065-4890-b07c-cd0be6016920" />

**Reply by Juanescacha** [reply]
> the max height is 20 even when 50 is specified?
> 
> <img alt="image" width="1274" height="138" src="https://private-user-images.githubusercontent.com/1082650/532618070-6436bdaf-a065-4890-b07c-cd0be6016920.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzY4MTUxNzMsIm5iZiI6MTc3NjgxNDg3MywicGF0aCI6Ii8xMDgyNjUwLzUzMjYxODA3MC02NDM2YmRhZi1hMDY1LTQ4OTAtYjA3Yy1jZDBiZTYwMTY5MjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI2MDQyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNjA0MjFUMjM0MTEzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjNlNDc2ZjM3NTA1MzM5ZWQ1MzgxYjM2NTM1ZTNkMzMxN2MwZDkyYTMwYjhjODRjYzUyYTUwMGRlZjE2ZmVmOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmcmVzcG9uc2UtY29udGVudC10eXBlPWltYWdlJTJGcG5nIn0.xK5v82D1JrCm-nZI_qJPs4tbqnuoRcX_58VEX19_cPo">

having the same issue, for the moment my workaround is to apply `style="height:50px"`
